Document how to upload larger files via front-end (#5960)
aces/Loris: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md

# Handling Large File Uploads

> In order to upload larger files (> 2 MB) for LORUS users, you'll need to configure

Reviewer: ```suggestion
In order to upload larger files (> 2 MB) for LORIS users, you'll need to configure
```

> In order to upload larger files (> 2 MB) for LORIS users, you'll need to configure

Reviewer: ```suggestion
In order for users to upload large files (> 2 MB), you'll need to configure
```

This wording is probably better

Apache's `php.ini` configuration file. 

> Recommended sample values appropriate for compressed scans not exceeding 500M in size are: 

Reviewer 2: ```suggestion
Recommended sample values appropriate for compressed scans not exceeding 500MB in size are: 
```

Reviewer: @johnsaigle ping.. I tried to click "Commit" on github so I can merge this PR but I don't have permission on the branch.
